tk-demos/widget

```diff
@@ -1,6 +1,6 @@
 #!/usr/local/bin/perl -w
 
-require 5.004;
+require 5.006;
```

Comment:
surprisingly, the tests were borrowed from perl/Tk, and I intentionally have not edited anything that is irrelevant to test passing

Reply:
So, do you want me to update the patch to not change these files?

Reply:
yes

Reply:
I removed the commit bumping the required Perl version.  Is that all you'd like changed?